README.md

```diff
@@ -31,72 +31,582 @@ _Kev_ reduces the need for Kubernetes expertise in the team. The generated Kuber
 
-If you want to point _Kev_ at set of alternative compose files, simply pass them in with `-f` flag. Multiple source compose files can be specified by providing `-f` flag multiple times.
+Also, it creates a default `dev` environment and its Compose override file.  
 
-And, to individually control the configuration of your project for deployments in separate target environments, just specify their respective names with `-e` flag, e.g.:
+Here's another example. It uses an alternate `docker-compose` file with `stage` + `prod` environments:
 
 ```sh
 kev init -f my-docker-compose.yaml -e stage -e prod
 ```
 
-If no environment has been specified, a default one will be named `dev`.
+It makes use of,
+- `-f` flag, to specify an alternate filename.
+- `-e` flags, to specify different deployment environments.
 
-Once the project has bootstrapped you should see a few new files added to your project tree, similar to the list below:
+Creating the files below in your project directory:
 
 ```sh
-├── docker-compose.kev.stage.yaml # stage environment override
-├── docker-compose.kev.prod.yaml  # prod environment override
-├── kev.yaml                      # kev project manifest
+├── docker-compose.kev.stage.yaml       # stage Compose environment override file
+├── docker-compose.kev.prod.yaml        # prod Compose environment override file
+├── kev.yaml                            # kev project manifest
 ├── ...
 ```
 
-Your project is now ready and environment specific configuration files generated. _Kev_ will infer useful configuration details already present in the source compose file(s). If this is not possible it'll assume sensible defaults. Users have an easy way to manually alter values directly in the environment specific override file as and when required.
+Here's what happened, _Kev_,
+- Inferred the configuration details already present in your compose Kubernetes deployment sources.
+- Assigned sensible defaults for any config it couldn't infer.
+- Created Compose overrides files for the `stage` and `prod` environments.
+
+That's it, _Kev_ is now bootstrapped and ready!
 
-### Render K8s manifests
+From now on it can,
+- Detect edits in your source compose file.
+- Apply any related config changes to your compose environment overrides.
+- Generate deployment manifests.   
 
-In order to deploy your project application to Kubernetes you will need to supply _something_ K8s can understand. Currently, _Kev_ only supports Docker Compose conversion to native Kuberentes manifests. The Community might add additional output formats at later stages. See our [Roadmap][roadmap] for planned new features.
+You can now customise your deployment targets by altering values in the relevant Compose environment override file. 
 
-To render manifests, simply run the following command from your project root:
+### render
```

Comment:
`render` -> `Render K8s manifests`

Reply:
Similar rationale to `init` as explained above.